.travis.yml

```diff
@@ -54,7 +54,7 @@ before_script:
 
 script:
   - py.test -n 4 --cov=mtpy --cov-report term:skip-covered tests/core
-  - MTPY_TEST_COMPARE_IMAGE=True py.test --cov=mtpy --cov-append --cov-report term:skip-covered tests/imaging  # save and compare images
+  - MTPY_TEST_COMPARE_IMAGE=True; py.test --cov=mtpy --cov-append --cov-report term:skip-covered tests/imaging  # save and compare images
```

Comment:
this ';' cannot be here, we want the image test to run with env variable MTPY_TEST_COMPARE_IMAGE=True